WordPress/gutenberg
# Image: Fix caption link wrapping in wide/full alignment

Fixes: #67206 

## What?
Changes the width CSS property of links within wide and full-width image block captions to prevent caption text from splitting across lines.

## Why?
Currently, when a caption in a wide or full-width Image Block contains a link, the linked text appears on a separate line from the rest of the caption text. This creates an undesirable visual break in what should be continuous caption text.

This occurs due to the `width: 100%` CSS property being applied to links within wide/full-width image blocks, forcing them to take up the full width and create a line break.

## Testing Instructions
1. Open the block editor with a block theme (e.g., Twenty Twenty-Five)
2. Insert an Image Block and add an image
3. Add a caption like "Photo Credit: Nature is Amazing Photo Collection"
4. Select "Nature is Amazing Photo Collection" and add a link
5. Test with Wide Width alignment: Caption should now appear on one line


## Screenshots 
![9](https://github.com/user-attachments/assets/d9e05a9a-2f92-4608-a69c-353f85a7929e)
![10](https://github.com/user-attachments/assets/36693e99-8c25-4a0a-907c-a417b42e2602)
